com.zen.browser.flatpak.yml

```diff
@@ -0,0 +1,66 @@
+app-id: com.zen.browser
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '22.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --env=GTK_PATH=/app/lib/gtkmodules
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+  - --talk-name=org.gnome.SessionManager
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=com.zen.browser.*
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - tar -xvf zen.linux.tar.bz2
+      - mv zen/ bin/
+
+      - install -d /app/bin
+      - install -d /app/bin/distribution
+      - install -d /app/share/icons
+      - install -d /app/share/metainfo
+      - install -d /app/share/applications
+
+      - cp -r bin/* /app/bin
+      - cp -r icons/* /app/share/icons
+      - cp launch-script.sh /app/bin/launch-script.sh
+      - cp -r policies.json /app/bin/distribution/policies.json
+
+      - cp com.zen.browser.metainfo.xml /app/share/metainfo/com.zen.browser.metainfo.xml
+      - cp com.zen.browser.desktop /app/share/applications/com.zen.browser.desktop
+
+      - chmod +x /app/bin/launch-script.sh
+    sources:
+      - type: file
+        url: https://github.com/zen-browser/desktop/releases/download/latest/zen.linux.tar.bz2
+
+      - type: file
+        path: ./policies.json
+
+      - type: file
+        path: ./launch-script.sh
+
+      - type: file
+        path: ./com.zen.browser.metainfo.xml
+      - type: file
+        path: ./com.zen.browser.desktop
+
+      - type: dir
+        path: ./icons/
+        dest: icons
```

Comment:
Don't use dir sources. You don't need icons of every size. Just install the highest sized PNG or install a SVG if present.